libcodechecker/generic_package_context.py

```diff
@@ -52,16 +40,33 @@ def __init__(self, package_root, pckg_layout, cfg_dict):
         self.__package_git_hash = None
         self.__analyzers = {}
 
+        self.logger_bin = None
+        self.logger_file = None
+        self.logger_compilers = None
+
+        # Get package specific environment variables.
+        self.set_env(env_vars)
+
         self.__set_version()
         self.__populate_analyzers()
 
-        Context.__instance = self
-
     def set_env(self, env_vars):
         """
         Get the environment variables.
         """
-        super(Context, self).set_env(env_vars)
+        self._package_root = os.environ.get(env_vars['env_package_root'])
+
+        self._codechecker_workspace = os.environ.get('codechecker_workspace')
+
+        try:
+            with open(self.checkers_severity_map_file) as severity_file:
+                self._severity_map = json.load(severity_file)
+        except (IOError, ValueError):
+            LOG.warning(self.checkers_severity_map_file + " doesn't exist or "
```

Comment:
Reformat this as a `"{0} doesn't exist...".format(self.checkers_...)` so that there aren't so many linebreaks.